FIRO_TSEnsembles/src/test/java/SqliteDatabaseTest.java

```diff
@@ -64,6 +64,20 @@ public void testMetricCollectionAsTimeSeries_TwoStep() throws Exception{
         }
     }
 
+    @Test
+    public void testDeleteAllEnsembles() throws Exception {
+        String sourceData = "src/test/resources/database/ResSimTest_20200101.db";
+        String copiedData = "C:\\Temp\\testdb_delete.db";
+        File sourcefile = new File(sourceData);
+        File copiedFile = new File(copiedData);
+        copyFileUsingStream(sourcefile,copiedFile);
+
+        SqliteDatabase db = new SqliteDatabase(copiedData, SqliteDatabase.CREATION_MODE.CREATE_NEW_OR_OPEN_EXISTING_UPDATE);
+        db.deleteAllEnsemblesFromDB();
+        Integer testVal = db.getEnsembleTimeSeriesIDs().size();
+        assertEquals(testVal, 0);
```

Comment:
Minor suggestion: The first argument to assertEquals  should be the expected value: 0,  the second argument is the actual.

Reply:
@ktarbet I have updated the test to show correct order (expected, actual) in `assertEquals`.